scissors/src/main/java/net/skinsrestorer/scissors/skin/SkinSection.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * SkinsRestorer
+ * Copyright (C) 2024  SkinsRestorer Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.skinsrestorer.scissors.skin;
+
+import lombok.Getter;
+import lombok.ToString;
+import net.skinsrestorer.scissors.MapHelpers;
+import net.skinsrestorer.scissors.RectangleSection;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Getter
+@ToString
+public enum SkinSection {
+    // Head base
```

Comment:
Maybe we could also add a link to the 2D schematic here and tell that it is from body parts starting from left top working zig zag to bottom right

Reply:
Sure. Btw in technical terms it's called an UV Map. It's what developers use to apply 2D textures onto 3D objects.
https://en.wikipedia.org/wiki/UV_mapping